### PR TITLE
Add migration guides for latest apis

### PR DESCRIPTION
**Pull request by natalieparellano**
Signed-off-by: Natalie Arellano <narellano@vmware.com>

### CODE REVIEW

File: content/docs/reference/spec/buildpack-api.md [hunk 1]
```diff
@@ -163,6 +163,27 @@ The schema is as follows:
...
     Clears user-defined environment variables when `true` on executions of `bin/detect` and `bin/build`.
+    
+    - **`homepage`** _(string, optional)_\
+    Buildpack homepage.
+    
+    - **`description`** _(string, optional)_\
+    A short description of the buildpack.
+    
+    - **`keywords`** _(string(s), optional)_\
+    Keywords to help locate the buildpack. These can be useful if publishing to the [Buildpack Registry](https://registry.buildpacks.io/).
```
Comment (natalieparellano) [inline content/docs/reference/spec/buildpack-api.md:174]: Added a few things that were missing from previous apis.

File: content/docs/reference/spec/migration/buildpack-api-0.6-0.7.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Buildpacks may write Software Bill of Materials (SBOM) files describing build- or run-time dependencies. These files must use the `application/vnd.cyclonedx+json`, `application/spdx+json`, or `application/vnd.syft+json` media types (a buildpack may output SBOM files in multiple formats). Files may be written to the following locations:
+
+* `<layers>/<layer>.sbom.<ext>` - for describing dependencies associated with a layer
+  * Example (launch layer): libraries that were included in an output compiled binary (e.g., `go` libraries in a `go` binary)
+  * Example (build layer): tools like a package manager
+* `<layers>/launch.sbom.<ext>` - for describing run-time dependencies not associated with a layer
+  * Example: dependencies installed in the `/workspace` directory
+* `<layers>/build.sbom.<ext>` - for describing build-time dependencies not associated with a layer
+  * Example: build time configuration
```
Comment (natalieparellano) [inline content/docs/reference/spec/migration/buildpack-api-0.6-0.7.md:21]: We could probably have a whole docs section on SBOM stuff. This seemed like the minimum information needed for now.

**Review by samj1912 — APPROVED** [review]

File: content/docs/reference/spec/migration/buildpack-api-0.6-0.7.md (hunk 2 above)
Comment (aemengo) [reply content/docs/reference/spec/migration/buildpack-api-0.6-0.7.md:21]: Looked plenty sufficient to me.

**Review by aemengo — APPROVED** [review]

File: content/docs/reference/spec/migration/platform-api-0.7-0.8.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Platform Operator
+
+### New standardized SBOM format
+
+Buildpacks implementing Buildpack API 0.7+ may output write Software Bill of Materials (SBOM) files describing build- or run-time dependencies. These files must use the `application/vnd.cyclonedx+json`, `application/spdx+json`, or `application/vnd.syft+json` media types (a buildpack may output SBOM files in multiple formats). Files may be written to the following locations:
+                                                      
+* `<layers>/<buildpack-id>/<layer>.sbom.<ext>` - for describing dependencies associated with a layer
+* `<layers>/<buildpack-id>/launch.sbom.<ext>` - for describing run-time dependencies not associated with a layer
+* `<layers>/<buildpack-id>/build.sbom.<ext>` - for describing build-time dependencies not associated with a layer
```
Comment (aemengo) [inline content/docs/reference/spec/migration/platform-api-0.7-0.8.md:19]: I noticed that there's no mention of `<layers>/<buildpack-id>/cache.sbom.<ext>`. Not sure if that was intentional.

Comment (natalieparellano) [reply content/docs/reference/spec/migration/platform-api-0.7-0.8.md:19]: Right, that is just a lifecycle implementation detail and is not spec'd.

**Comment by natalieparellano** [comment]
Added 3c4820a to clarify that the launch-level and build-level boms are not restored.

**Review by jromero — APPROVED** [review]

**Pull request merged**